source/common/router/rds_subscription.cc

```diff
@@ -50,6 +50,7 @@ void RdsSubscription::parseResponse(const Http::Message& response) {
   resources[0].set_name(route_config_name_);
   callbacks_->onConfigUpdate(resources);
   version_info_ = Envoy::Config::Utility::computeHashedVersion(response_body);
+  stats_.version_.set(HashUtil::xxHash64(version_info_));
```

Comment:
i would just hash the response body again here

Reply:
Please avoid computing the hash twice. Just return a tuple or something.

Reply:
But +1 for using same hash.